project.clj

```diff
@@ -1,22 +1,24 @@
-(defproject crypto-password "0.2.0"
+(defproject crypto-password "0.2.1"
   :description "Library for securely hashing passwords"
   :url "https://github.com/weavejester/crypto-password"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.5.1"]
+  :dependencies [[org.clojure/clojure "1.10.0"]
```

Comment:
No need to change the minimum Clojure version, as far as I can see?

Reply:
you are right